### ISSUE TITLE
Click on group marker on overview map doesn't show public group info

### ISSUE THREAD

**Issue by djahnie** [yunity/karrot-frontend#995]
When clicking the marker the group name shows, when clicking the name this happens:

It should take the user to the public group info, instead it tries to access to group. If the user is not a member of the group already this leads to the banana error 'Ooops, you are not a member of the group'.

**Comment by trmendes** [comment]
I'm not sure if I understood it but to double check:

If I click on a group I'm not a member all I see is banana error and this is not what it should happen.

The suggestion to fix it would be If I click on a group I'm not a member I should see the public group info instead.

Is that right?

**Comment by djahnie** [comment]
Yes, that is how I meant it. Do you want to work on this, @trmendes?
